src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -66,7 +71,14 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      */
     mapping(address => mapping(bytes32 => bool)) public delegationApproverSaltIsSpent;
 
-    IEigenPodManager public immutable eigenPodManager;
+    /// @notice Mapping: hash of withdrawal inputs, aka 'withdrawalRoot' => whether the withdrawal is pending
+    mapping(bytes32 => bool) public pendingWithdrawals;
+
+    /// @notice Mapping: staker => cumulative number of queued withdrawals they have ever initiated.
+    /// @dev This only increments (doesn't decrement), and is used to help ensure that otherwise identical withdrawals have unique hashes.
+    mapping(address => uint96) public numWithdrawalsQueued;
```

Comment:
can we add cumulative to the name here? confusing with whether this is the number of withdrawals queued *at the time of calling*

Reply:
good suggestion, done https://github.com/Layr-Labs/eigenlayer-contracts/pull/222/commits/3ee27af5e0379de1ca0c486a923e4a6c9687da8f